test_scripts/Defects/6_1/3330_common.lua

```diff
@@ -0,0 +1,54 @@
+---------------------------------------------------------------------------------------------------
+-- common module for 3330 issue
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local utils = require('user_modules/utils')
+local commonRC = require('test_scripts/RC/commonRC')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Common Variables ]]
+local m = {}
+m.preconditions = commonRC.preconditions
+m.start = commonRC.start
+m.registerAppWOPTU = commonRC.registerAppWOPTU
+m.activateApp = commonRC.activateApp
+m.postconditions = commonRC.postconditions
+
+--[[ Common Functions ]]
+function m.rpcAllowed(isIdParam)
+  local seatParams = utils.cloneTable(commonRC.getModuleControlData("SEAT"))
+  if isIdParam ~= true then seatParams.seatControlData.id = nil end
```

Comment:
I would explicitly define this flag since this is the main aim of the test.
In this way looking into script it will be clear the test data and expected result.